core/finality-grandpa/Cargo.toml

```diff
@@ -27,6 +27,7 @@ network = { package = "substrate-network", path = "../network" }
 srml-finality-tracker = { path = "../../srml/finality-tracker" }
 fg_primitives = { package = "substrate-finality-grandpa-primitives", path = "primitives" }
 grandpa = { package = "finality-grandpa", version = "0.9.0", features = ["derive-codec"] }
+state-machine = { package = "substrate-state-machine", path = "../state-machine" }
```

Comment:
We should put 'StorageProof' in a different crate (trie or primitive) to avoid those new dependencies.

Reply:
It's nice to have it as part of the `proving_backend` module so that the internals can stay private/opaque.

Reply:
It is not re-exported from `client::light::fetcher`.